managing_providers/_topics/configuration_management_providers.md

```diff
@@ -15,9 +15,10 @@ Configuration management providers must be added individually to
 
 ## IBM Terraform provider
 
-Terraform is an Open Source software that is developed by HashiCorp that enables predictable and consistent provisioning of many cloud platforms, classic infrastructure, and virtual private cloud (VPC) infrastructure resources by using a high-level scripting language. You can use Terraform to automate your cloud resource provisioning, rapidly build complex, multi-tier cloud environments, and enable Infrastructure as Code (IaC). {{ site.data.product.title }} provides functionality to integrate with Terraform and take advantage of features. This includes:
+{{ site.data.product.title_short }} includes an IBM Terraform provider that provides functionality to integrate with the [Managed Services component](https://www.ibm.com/support/knowledgecenter/SSFC4F_2.2.0/cam/cam_intro.html). Managed services uses open source Terraform to manage and deliver cloud infrastructure as code. Terraform is an Open Source software that is developed by HashiCorp that enables predictable and consistent provisioning of many cloud platforms, classic infrastructure, and virtual private cloud (VPC) infrastructure resources by using a high-level scripting language. You can use Terraform to automate your cloud resource provisioning, rapidly build complex, multi-tier cloud environments, and enable Infrastructure as Code (IaC).
```

Comment:
The audience for this doc on upstream might not know the context of the CloudPak, so this line might need some clarification of Managed Services Component.  Thoughts?

```suggestion
{{ site.data.product.title_short }} includes an IBM Terraform provider that provides functionality to integrate with the [Managed Services component](https://www.ibm.com/support/knowledgecenter/SSFC4F_2.2.0/cam/cam_intro.html) of the IBM Cloud Pak for Multicloud Management. Managed services uses open source Terraform to manage and deliver cloud infrastructure as code. Terraform is an Open Source software that is developed by HashiCorp that enables predictable and consistent provisioning of many cloud platforms, classic infrastructure, and virtual private cloud (VPC) infrastructure resources by using a high-level scripting language. You can use Terraform to automate your cloud resource provisioning, rapidly build complex, multi-tier cloud environments, and enable Infrastructure as Code (IaC).
```